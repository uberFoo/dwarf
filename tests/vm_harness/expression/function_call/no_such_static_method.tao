struct Point {
    x: int,
    y: int,
}

impl Point {
    fn new(x: int, y: int) -> Point {
        Point { x: x, y: y }
    }

    fn x(self) -> int {
        self.x
    }

    fn y(self) -> int {
        self.y
    }
}

fn main() {
    let p = Point::default();
}
