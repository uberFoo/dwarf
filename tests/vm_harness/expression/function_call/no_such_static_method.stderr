Unable to compile program:
[31mError:[0m no such method
    [38;5;246mâ•­[0m[38;5;246mâ”€[0m[38;5;246m[[0m no_such_static_method:21:13 [38;5;246m][0m
    [38;5;246mâ”‚[0m
 [38;5;246m21 â”‚[0m [38;5;249m [0m[38;5;249m [0m[38;5;249m [0m[38;5;249m [0m[38;5;249ml[0m[38;5;249me[0m[38;5;249mt[0m[38;5;249m [0m[38;5;249mp[0m[38;5;249m [0m[38;5;249m=[0m[38;5;249m [0m[31mP[0m[31mo[0m[31mi[0m[31mn[0m[31mt[0m[31m:[0m[31m:[0m[31md[0m[31me[0m[31mf[0m[31ma[0m[31mu[0m[31ml[0m[31mt[0m[31m([0m[31m)[0m[38;5;249m;[0m
 [38;5;240m   â”‚[0m             [31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”¬[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m  
 [38;5;240m   â”‚[0m                     [31mâ•°[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m in this invocation: [33mdefault[0m
[38;5;246mâ”€â”€â”€â”€â•¯[0m

